code/modules/reagents/reagent_containers.dm

```diff
@@ -126,7 +126,7 @@
 		to_chat(user, "<span class='notice'>[target] is empty.</span>")
 		return 1
 
-	if(reagents && !reagents.get_free_space())
+	if(reagents && !REAGENTS_FREE_SPACE(reagents))
```

Comment:
Pretty sure every `if(src.reagents)` in this file is unnecessary since it creates reagents in Initialize and I don't think there's anything that deletes that datum. 

Probably need the `if(target.reagents)` checks tho.